src/include/planner/operator/ddl/logical_alter.h

```diff
@@ -2,21 +2,42 @@
 
 #include "binder/ddl/bound_alter_info.h"
 #include "logical_ddl.h"
+#include "planner/operator/ddl/base_alter.h"
 
 namespace kuzu {
 namespace planner {
 
+struct LogicalAlterPrintInfo final : OPPrintInfo {
```

Comment:
After reading through the code I suggest do the following.

```
struct LogicalAlterPrintInfo final : OPPrintInfo {
common::AlterType alterType;
std::string tableName;
binder::BoundExtraAlterInfo* info;

std::string toString() ....
}

struct AlterPrintInfo final : OPPrintInfo {
LogicalAlterPrintInfo info

std::string toString() const override {return info->toString();};
```
And remove BaseLogicalAlterPrintInfo.

If this make sense, propagate this change to CreateTable as well